bin/console

```diff
@@ -1,14 +1,18 @@
+# frozen_string_literal: true
```

Comment:
`# frozen_string_literal: true` needs to come after `#!/usr/bin/env ruby`, and the file should be made executable.